.drone.star

```diff
@@ -2450,7 +2450,8 @@ def buildGithubCommentForBuildStopped(suite, alternateSuiteName):
         },
     }]
 
-def githubComment():
+def githubComment(alternateSuiteName):
+    prefix = "Results for <strong>%s</strong>" % alternateSuiteName
```

Comment:
IMO this is going to also prefix the other failure comments, like ":boom: Acceptance tests pipeline <strong>%s</strong> failed. The build has been cancelled."

And those already have `alternateSuiteName` embedded in the text.

But we can can merge this - it does not break anything - and then cleanup whichever comments have duplication.